### PR TITLE
Guidelines and templates

### PR DESCRIPTION
**Pull request by cpswan**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (murali-shris) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: should we add app field? Most of our functionality will be tested through apps

Comment (cpswan) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: Sounds good. Can you please do a PR against https://github.com/atsign-foundation/archetype/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

Comment (cpswan) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: Actually - I'll have a go at this myself, and if you don't like it, then we can iterate

Comment (cpswan) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: Apps have been added now by https://github.com/atsign-foundation/at_tools/pull/62/commits/73919615021d764091e97fbbda535d2629975278 so are we good to go with this?

**Comment by cpswan** [comment]
@murali-shris I'm still waiting on the green light to merge this

**Review by murali-shris — APPROVED** [review]

**Comment by murali-shris** [comment]
> @murali-shris I'm still waiting on the green light to merge this

apologies for the delay.

**Pull request merged**
